changeset:   4394:42012a6afb5b
branch:      SDL-1.2
user:        Sam Lantinga <slouken@libsdl.org>
date:        Thu Dec 17 14:33:00 2009 +0000
summary:     Merged SDL 1.3 revision 5424, fixing a crash in the joystick code on recent kernels.

diff -r 9afe12fb4c41 -r 42012a6afb5b src/joystick/linux/SDL_sysjoystick.c
--- a/src/joystick/linux/SDL_sysjoystick.c	Mon Dec 14 22:41:31 2009 +0000
+++ b/src/joystick/linux/SDL_sysjoystick.c	Thu Dec 17 14:33:00 2009 +0000
@@ -700,7 +700,7 @@
 				continue;
 			}
 			if ( test_bit(i, absbit) ) {
-				int values[5];
+				int values[6];
 
 				if ( ioctl(fd, EVIOCGABS(i), values) < 0 )
 					continue;

